examples/vision/attention_mil_classification.py

```diff
@@ -0,0 +1,575 @@
+"""
+Title: Classification using Attention-based Deep Multiple Instance Learning (MIL).
+Author: [Mohamad Jaber](https://www.linkedin.com/in/mohamadjaber1/)
+Date created: 2021/08/16
+Last modified: 2021/09/09
+Description: MIL approach to classify bags of instances and get their individual instance score.
+"""
+"""
+Suppose there is a locked door and several key chains with each chain containing few keys.
```

Comment:
This analogy does not make the problem easy to understand IMO. The explanation in the introduction section is clearer.

Reply:
Removed it.